### PR TITLE
[FEAT] Add upcoming items

### PR DESCRIPTION
**Pull request by adamhannigan**
# Description

This PR adds the UI elements for the upcoming items, including decorations and special events.

Red Envelope + Mankei Neko is is accordance with Chinese NYE 🚀 

Screen Shot 2023-01-11 at 3 41 24 pm

### CODE REVIEW

File: src/features/game/types/collectibles.ts [hunk 1]
```diff
@@ -66,4 +67,13 @@ export const GOBLIN_BLACKSMITH_ITEMS: Record<
     supply: 535,
     boost: "+1 Blueberry",
   },
+  "Maneki Neko": {
+    description: "The beckoning cat. Pull it's arm and good luck will come",
+    ingredients: {
+      Gold: new Decimal(5),
+      "Red Envelope": new Decimal(50),
```
Comment (adamhannigan) [inline src/features/game/types/collectibles.ts:74]: This is just a placeholder right now - will need to design this mechanic + drop rate

Comment (mGranTY) [reply src/features/game/types/collectibles.ts:74]: If its following the proposal, we can reuse the money tree modal to display the Red Evelope drop inside a chest reward.

Comment (adamhannigan) [reply src/features/game/types/collectibles.ts:74]: We've got a few technical constraints at the moment - the first one is that trees don't support random drop/reward mechanics yet 😢 We may need to reuse the crop reward system, unless we get time to focus on a tree drop system

Comment (mGranTY) [reply src/features/game/types/collectibles.ts:74]: This don't support item rewards?

https://github.com/sunflower-land/sunflower-land/blob/5da412a6172d93d3f8aa5a564076203d50a95d5a/src/features/game/events/landExpansion/collectTreeReward.ts#L54

Comment (adamhannigan) [reply src/features/game/types/collectibles.ts:74]: My bad, I didn't even know we had that. Thank you to whoever implemented it 🙏

File: metadata/metadata.ts [hunk 2]
```diff
@@ -2912,4 +2912,41 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
...
+  "Ayam Cemani": {
+    description: "The rarest chicken in Sunflower Land",
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/445.gif",
+    attributes: [{ trait_type: "Purpose", value: "Decoration" }],
+  },
+  "Maneki Neko": {
+    description:
+      "The beckoning cat. Pull it's arm and good luck will come. A special event item from Chinese New Years!",
```
Comment (spencerdezartsmith) [inline metadata/metadata.ts:2924]: I'm not sure but I don't think Chinese New Year is pluralised.

File: metadata/metadata.ts [hunk 3]
```diff
@@ -2912,4 +2912,41 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
...
+  "Cyborg Bear": {
+    description: "Hasta la vista, bear",
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/1222.png",
+    attributes: [{ trait_type: "Purpose", value: "Decoration" }],
+  },
+  "Red Envelope": {
+    description:
+      "Wow, you are lucky! An item from the Chinese New Years special event.",
```
Comment (spencerdezartsmith) [inline metadata/metadata.ts:2946]: Again, I think it should read `from Chinese New Year`

**Review by spencerdezartsmith — APPROVED** [review]
Just some grammar stuff. Love these new items! 🐔

**Pull request merged**